### ISSUE TITLE
Fastpath for CFDiskArray with scale = 1 and offset = 0

### ISSUE THREAD

**Issue by gdkrmr** [meggart/DiskArrayTools.jl#13]
The new DataCube v3 contains scale and offset for all variables (they are all 1 and 0 respectively [1], they are still missing for some variables and this is a bug). Is it worth implementing a fast path for this or is the performance impact so small that we do not care?

[1] http://data.rsc4earth.de/EarthSystemDataCube/v3.0.1/esdc-8d-0.25deg-256x128x128-3.0.1.zarr/latent_energy/.zattrs

**Comment by meggart** [comment]
Yes, we definitely should. How exactly do you handle missing values in v3? I don't see a `missing_value` attribute in the metadata, does this mean you simply interpret NaNs as missing and assume all variables are floating point?

**Comment by meggart** [comment]
See #14

**Comment by gdkrmr** [comment]
> How exactly do you handle missing values in v3?

We just use NaNs. Should we declare `missing_value = NaN` explicitly?

**Comment by meggart** [comment]
> Should we declare missing_value = NaN explicitly?

It depends on what you expect inside YAXArrays. If you want these NaNs to be translated into `missing`s then yes, you should declare this. If you are happy to work with pure Float arrays and treat NaNs as missings then leave it as it is.
